sql_schema/BookMyShow.sql

```diff
@@ -0,0 +1,50 @@
+CREATE DATABASE Movies;
+
```

Comment:
Provide the option to book seats while booking

Reply:
CREATE TABLE Seats(
SeatID INT PRIMARY KEY,
SeatNo VARCHAR(5) NOT NULL,
BookingID INT NOT NULL,
FOREIGN KEY (BookingID) REFERENCES Booking(BookingID));